test/epgsql_tests.erl

```diff
@@ -959,7 +959,18 @@ range_type_test(Module) ->
       Module,
       9.2,
       fun(_C) ->
-          check_type(Module, int4range, "int4range(10, 20)", <<"[10,20)">>,
+          check_type(Module, int4range, "int4range(10, 20)", {10, 20},
+                     [{1, 58}, {-1, 12}, {-985521, 5412687}, {minus_infinity, 0},
+                      {984655, plus_infinity}, {minus_infinity, plus_infinity}])
+      end,
+      []).
+
+range8_type_test(Module) ->
+    with_min_version(
+      Module,
+      9.2,
+      fun(_C) ->
+          check_type(Module, int8range, "int8range(10, 20)", {10, 20},
```

Comment:
It would be nice to test values greater than int32. Ideally - upper and lower bounds.